build.sbt

```diff
@@ -2092,6 +2093,7 @@ lazy val `std-image` = project
       result
     }.value
   )
+  .dependsOn(`std-base` % "provided")
```

Comment:
It's kind of weird that the classifier is here but I can see we do the same with `std-table`. Will need to investigate separately